pulp_rpm/app/models/package.py

```diff
@@ -185,7 +185,7 @@ class Package(Content):
     # not part of createrepo_c metadata
     is_modular = models.BooleanField(default=False)
 
-    repo_key_fields = ('name', 'epoch', 'version', 'release', 'arch')
+    repo_key_fields = ('name', 'epoch', 'version', 'release', 'arch', 'location_href')
```

Comment:
@dralley , @daviddavis? are you ok with that change until we figure out the relative path problem?
Or do you prefer to use `location_href` only?

Reply:
It's probably OK for now.

Reply:
@pavelpicka can you add a comment, that createrepo_c treats src and nosrc as src, so 2 packages like pkg-1.src.rpm and pkg-1.nosrc.rpm won't be saved in a repo version unless we add location_href among repo_key fields

Reply:
Comment the explanation for easier understanding why we are adding it.